src/PluginBase.php

```diff
@@ -24,6 +24,18 @@ abstract class PluginBase implements PluginInterface, EventSubscriberInterface
 
 	protected $io;
 
+	protected $configPath;
+
+	protected $config;
```

Comment:
And this? ;)

Also: can we specify a type?

Reply:
Sadly not if we want to support 7.2 :(

Reply:
**mumbles**

`/** @var <type> */` then? pretty please? ;-)